scss/book-list-detail.scss

```diff
@@ -1,12 +1,49 @@
 .master-container{
   display: flex;
-  max-width: 1366px;
-  width: 100%;
+  height: 768px;
+  width: 1366px;
   background-color: $wild-sand;
-  justify-content: center;
-  align-items: center;
+
+  .navbar-container {
+    position: fixed;
+    display: flex;
+    top: 0;
+    left: 0;
+    border-top: 6px solid $cerulean; 
+    height: 70px;
+    width: 100%;
+    max-width: 1366px;
+    background-color: $white;
+    box-shadow: 0 4px 10px 0 $black-rgba;
+    align-items: center;
+    align-content: center;
```

Comment:
Is this rule necessary?

Reply:
align-content isnt, but align-items is necessary so that the wolox logo and the 'logout' text are aligned correctly